server/src/main/java/org/opensearch/index/mapper/DateFieldMapper.java

```diff
@@ -457,30 +461,39 @@ public Query rangeQuery(
             @Nullable DateMathParser forcedDateParser,
             QueryShardContext context
         ) {
-            failIfNotIndexedAndNoDocValues();
+            failIfNotIndexed();
```

Comment:
Are we not supporting range query on just doc values anymore?

Reply:
whoops that was an unintended change, reverted that.